packages/test-utils/tsconfig.lib.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
```

Comment:
Do you need `outDir`?

Reply:
Actually don't think so, this is just the generic template by Nx.